File: Lib/Telerik/Examples/QuickStart/ChartView/Appearance/PaletteListDataItem.cs
using System;
using Telerik.WinControls.UI;
using Telerik.WinControls;

namespace Telerik.Examples.WinControls.ChartView.Appearance
{
    public class PaletteListDataItem : RadListDataItem
    {
        public static readonly RadProperty PaletteProperty = RadProperty.Register("Palette", typeof(ChartPalette), typeof(PaletteListDataItem), new RadElementPropertyMetadata(false));

        public ChartPalette Palette
        {
            get
            {
                return (ChartPalette)this.GetValue(PaletteListDataItem.PaletteProperty);
            }
            set
            {
                this.SetValue(PaletteListDataItem.PaletteProperty, value);
            }
        }
    }
}

File: Lib/Telerik/Examples/QuickStart/ChartView/Appearance/PaletteListDataItem.vb
Imports Telerik.WinControls.UI
Imports Telerik.WinControls

Namespace Telerik.Examples.WinControls.ChartView.Appearance
	Public Class PaletteListDataItem
		Inherits RadListDataItem
		Public Shared ReadOnly PaletteProperty As RadProperty = RadProperty.Register("Palette", GetType(ChartPalette), GetType(PaletteListDataItem), New RadElementPropertyMetadata(False))

		Public Property Palette() As ChartPalette
			Get
				Return CType(Me.GetValue(PaletteListDataItem.PaletteProperty), ChartPalette)
			End Get
			Set(ByVal value As ChartPalette)
				Me.SetValue(PaletteListDataItem.PaletteProperty, value)
			End Set
		End Property
	End Class
End Namespace
